src/winetricks

```diff
@@ -12292,6 +12292,81 @@ load_pngfilt()
 
 #----------------------------------------------------------------
 
+w_metadata powershell_core apps \
+    title="PowerShell Core" \
+    publisher="Microsoft" \
+    year="2024" \
+    media="download" \
+    file1="PowerShell-7.2.19-win-x64.msi" \
+    file2="PowerShell-7.2.19-win-x86.msi"
```

Comment:
Pleas swap these (x86 should be file1)

Reply:
Fixed by 509afca8a64d185256e28d9efd9e5e87bb71a6e3